qunit-dom/package.json

```diff
@@ -38,34 +47,13 @@
       "/dist/"
     ]
   },
-  "dependencies": {
```

Comment:
We need to include addon-shim here https://github.com/embroider-build/embroider/blob/main/packages/addon-shim/README.md

Reply:
we do not, because this isn't an _ember addon_, it's a vanilla package that non-ember projects can use.